cli/lib/kontena/machine/cloud_config/cloudinit.yml

```diff
@@ -19,27 +19,28 @@ write_files:
       fs.inotify.max_user_instances = 8192
 coreos:
   units:
-    - name: 00-eth.network
-      runtime: true
+    - name: 10-weave.network
+      mask: true
+    - name: 50-docker.network
+      mask: true
+    - name: 50-docker-veth.network
+      mask: true
+    - name: zz-default.network
       content: |
+        # default should not match virtual Docker/weave bridge/veth network interfaces
```

Comment:
We could also try the `Path=*` trick, but I dunno exactly.

Maybe something related to the already existing `--peer-interface`?
```ruby
match_interface = peer_interface.sub(/\d+/, '*')
```